examples/accounts/create-sub-account/Ballerina.toml

```
@@ -1,5 +1,5 @@
 [package]
-org = "dilanperera"
+org = "wso2"
```

I would use something like "twilio_samples", instead of wso2 as the org name :) WDYT?
```suggestion
org = "twilio_samples"
```

---

Sure. let's change it in that way

---

Changed with [ee8b23a](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/228/commits/ee8b23a322d096929298633c29e0b0a318529986).